modules/serverless-rn-1-7-0.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-7-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.7.0
+
+[id="new-features-1-7-0_{context}"]
+== New features
+* {ServerlessProductName} 1.7.0 is now Generally Available (GA) on {product-title} 4.3 and newer versions. In previous versions, {ServerlessProductName} was a Technology Preview.
+* {ServerlessProductName} now uses Knative Serving 0.13.2.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.13.2.
+* {ServerlessProductName} now uses Knative Serving Operator 0.13.2.
+* Knative Eventing is now available as a Technology Preview with {ServerlessProductName}. {ServerlessProductName} uses Knative Eventing 0.13.2.
+* Active/passive high availability (HA) is now available by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
+// TODO: explain the components of serving elsewhere in the docs, show a screenshot of the components in the web console, how to check status via CLI, etc.
++
+If you have installed a previous version of {ServerlessProductName}, after the KnativeServing custom resource (CR) is updated, the deployment will default to a HA configuration with `KnativeServing.spec.high-availability.replicas = 2`.
+* {ServerlessProductName} now supports the `trustedCA` setting in {product-title}'s cluster-wide proxy, and is now fully compatible with {product-title}'s proxy settings.
+
```

Comment:
We now support HTTPS using the wild-card certificate that's registered for Openshift Routes in general.

The user doesn't need to do anything to enable that, it will be "just there". The Knative Services will show "http://" URLs, but "https://" will work as well. Both will work.

On upgrade from 1.6.0, all routes will change to the new format so this is a breaking change.

Reply:
In our upgrade tests I've only seen failing the prober test but after the upgrade the other tests ran successfully. So I think it only breaks applications which saved the URL before the upgrade and are using it now, and not breaking applications that newly retrieved the URL after the upgrade. Is that correct? @markusthoemmes